### DISCUSSION TITLE
Work-itemy nie są zlinkowane z releasem kiedy build jest w innym projekcie niż kod

### DISCUSSION THREAD

**Discussion by roofiq** [AzureCommunityPL/group#219] [Pomoc]
Cześć!

Mam problem z linkowaniem work-itemów kiedy yaml build jest w innym projekcie niż kod (nie widać work-itemów w widoku classic releasów po zakończonym deploymenie). Poniżej wklejam opis jaki wrzuciłem również pod docsem microsoftu [https://docs.microsoft.com/en-us/azure/devops/pipelines/process/resources?view=azure-devops&tabs=schema](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/resources?view=azure-devops&tabs=schema).

 Jest tu wyraźnie napisane, że kiedy korzystamy z Resources to mamy dostęp do wszystkiego, w tym work-itemów. No ale jakoś to nie działa :D może znacie jakiś workaround? 

I have yaml definition in separate Azure DevOps project and my code in another one. In my yaml template, I am using resources repositories to set triggers and checkout my code repository.

Then I have classic release pipelines which are consuming my yaml build. All is fine, but in the linked document we can read, that using resources allow us to actually have full traceability of those consumed services, including work-items.
In my release pipeline, after deployment, I can't see any work-items connected to this build and they should be visible based on this doc.

The only solution I've found so far is to move yaml pipelines to my code repository. Could someone explain it to me? What is the best strategy of keeping those CI pipelines. I know, that preferably they should be near the code. But my project specification does not allow it and in the same time we need to have those work-items linked to release.

Dzięki za pomoc z tym!
Rafał

**Comment by wi5nia** [comment]
Czyli zeby dobrze zrozumiec scenariusz:
ProjektA: Work Itemy
ProjektB: kod + pipeline'y
?

**Reply by roofiq** [reply]
Projekt A kod i work itemy, projekt B pipeline :)

**Comment by JanuszNowak** [comment]
Brojekt B-> pipeliney yaml + classic release ? Integraca włączona "Report deployment status to Work" ? Może screen gdzie ci tego brakuje ?

**Comment by roofiq** [comment]
Definicja YAML'owa jest w projekcie B, ale już pipeline i classic release w projekcie B. Zapomniałem o tym ostatnim :)

Poniżej screen z przykładem.
Tutaj ten release jest w projekcie A, [core]-Api również w A ale definicja jest w B, gdzie przez resources repositories zaciągamy kod aplikacji z projektu A.
@JanuszNowak  integracja właczona.

![image](https://user-images.githubusercontent.com/44588684/108821271-a310fb80-75bd-11eb-836b-761e5297671c.png)

Wydaje mi się, że jedyną opcją jest dodanie yamla do projektu A, blisko kodu, który będzie ewentualnie zawierał jakieś templatki z projektu B.
